src/main/java/de/t0biii/music/domain/Music.java

```diff
@@ -79,12 +84,10 @@ private static void playSong(Player player, Main plugin) {
       }
       playingSong.put(player.getUniqueId(), sp);
 
-      if (plugin.getConfig().getBoolean("options.printSongTitel")) {
-        if (!sp.getSong().getTitle().isEmpty()) {
-          player.sendMessage(plugin.prefix + "§2Start Playing the Song:§a§l " + sp.getSong().getTitle());
-        } else {
-          player.sendMessage(plugin.prefix + "§2Start Playing a Song.");
-        }
+      String playingMessage = plugin.getConfig().getString("messages.playing");
+      if (!playingMessage.isEmpty()) {
+        player.sendMessage(plugin.prefix + 
+        		playingMessage.replaceAll("%song%",sp.getSong().getTitle()).replaceAll("&", "§"));
```

Comment:
maybe we want to check if the Title is Empty? 
https://i.tobias.bayern/lyiwCqOT.png

Reply:
Yeah, it actually could be a good idea, since while I thought of doing it in a way that if the admin knows the song has no title, then they just remove the `%song%` placeholder from there, but now I'm thinking about the random feature maybe it would be good to have a default.